3-4/3-4.js

```diff
@@ -1,7 +1,6 @@
-// Guessing
-var num = window.prompt("Enter a number");
+const num = window.prompt("Enter a number");
 if (num && parseInt(num, 10) <= 5 && parseInt(num, 10) >= 0) {
-	var random = Math.floor(6 * Math.random());
+	const random = Math.floor(6 * Math.random());
 	if (num == random) {
```

Comment:
Use `===` instead of `==` as much as possible.
[JavaScript Showdown: == vs === - codeburst](https://codeburst.io/javascript-showdown-vs-7be792be15b5)